src/content/blog/astro-development-with-ddev.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Astro development with DDEV"
+pubDate: 2023-11-30
+# modifiedDate: 2023-07-23
+summary: How to run Astro, Vite and other npm packages with DDEV
+author: Bernardo Martinez
+featureImage:
+    src: /img/blog/2023/10/astroonddev.png
+    alt: Astro image
+categories:
+  - Guides
+  - DevOps
+---
+
+Some projects might require you to add front-end tooling. Or the whole project might be based around client-side frameworks. DDEV provides an npm command out of the box, simplifying Docker development. I will be using this website as an example.
```

Comment:
I don't think this article is about DDEV having an `npm` command, lead with a stronger paragraph.